packages/openactive-broker-microservice/src/util/item-transforms.js

```diff
@@ -0,0 +1,91 @@
+const { partition, omit } = require('lodash');
+
+/**
+ * Inverts any items that have an `individualFacilityUse` property, so that the top-level `kind` is "IndividualFacilityUse"
+ *
+ * @param {{state: string, modified:any, kind: string, id: string, data: {individualFacilityUse: object}}[]} items
+ */
+function invertFacilityUseItems(items) {
+  const [invertibleFacilityUseItems, otherItems] = partition(items, (item) => item.data?.individualFacilityUse?.length > 0);
```

Comment:
```suggestion
  const [invertibleFacilityUseItems, otherItems] = partition(items, (item) => (item.data?.individualFacilityUse || []).length > 0);
```

Otherwise it's comparing `undefined` to `0` which apparently works as intended in this case but still

Reply:
+1